src/e2e/java/teammates/e2e/cases/e2e/BaseE2ETestCase.java

```diff
@@ -146,4 +153,49 @@ protected AdminHomePage loginAdmin() {
         return loginAdminToPage(createUrl(Const.WebPageURIs.ADMIN_HOME_PAGE), AdminHomePage.class);
     }
 
+    /**
+     * Deletes file with fileName from the downloads folder.
+     */
+    protected void deleteDownloadsFile(String fileName) {
+        String filePath = getTestDownloadsFolder() + fileName;
+        FileHelper.deleteFile(filePath);
+    }
+
+    /**
+     * Verifies downloaded file has correct fileName and contains expected content.
+     */
+    protected void verifyDownloadedFile(String expectedContent, String expectedFileName) {
+        try {
+            String filePath = getTestDownloadsFolder() + expectedFileName;
+            assertTrue(FileHelper.readFile(filePath).contains(expectedContent));
```

Comment:
Also, just checking one line may not be enough. Let's make `expectedContent` a list.